src/globus_sdk/experimental/globus_app/app.py

```diff
@@ -264,18 +323,25 @@ def _auth_params_with_required_scopes(
 
         return auth_params
 
-    def get_authorizer(self, resource_server: str) -> GlobusAuthorizer:
+    def get_authorizer(
+        self,
+        resource_server: str,
+        *,
+        skip_error_handling: bool = False,
+    ) -> GlobusAuthorizer:
         """
         Get a ``GlobusAuthorizer`` from the app's authorizer factory for a specified
         resource server. The type of authorizer is dependent on the app.
 
-        :param resource_server: the resource server the Authorizer will provide
-            authorization headers for
+        :param resource_server: the resource server for which the requested Authorizer
```

Comment:
The parameter lines are sentence-cased throughout this PR.

```suggestion
        :param resource_server: The resource server for which the requested Authorizer
```